webcrypto/types.go

```diff
@@ -6,6 +6,22 @@ import (
 	"github.com/dop251/goja"
 )
 
+// bitLength is a type alias for the length of a bits collection.
+type bitLength int
+
+// asByteLength returns the length of the bits collection in bytes.
+func (b bitLength) asByteLength() int {
```

Comment:
wouldn't it be better if this actually returns `byteLength`?

I did try locally and while this means that now at some places we need to cast it to `int` int also removed the need to cast it to byteLength in kind of the same amount of places.

Same for `byteLength`.

Reply:
I tried that too, but we do have to make a bunch casts as a result indeed, which felt a bit like it was defeating the purpose, and I was waiting to see what you thought 👍🏻 

I'll go ahead and change the return types then 🙇🏻